core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
that is all correct and the comment still applies.
wh_couch wants to get everything. it results in 
```[{conf,[federate]}]
   [{conf,[federate, {'type', <<"database">>}]}]
```
if a database is deleted in one node, the other node would be notified and the cached docs would be erased.

when there is no binding we don't apply the type/database.